components/AboutSection.js

```diff
@@ -0,0 +1,46 @@
+import AboutSection from './SectionTitle'
+export default function About () {
```

Comment:
Sorry, I didn't make it clear last time.  😭 
By changing the component name, I mean changing line 2 to
```
export default function AboutSection () {
```

Therefore, please change the import statement back to
```
import SectionTitle from './SectionTitle'
```

😎